COLLABORATOR_GUIDE.md

```diff
@@ -222,6 +222,25 @@ not used in other CI test runs (such as tests in the `internet` or `pummel`
 directories). It can also make sure tests pass when provided with a flag not
 used in other CI test runs (such as `--worker`).
 
+From the CI Job page, click "Build with Parameters" on the left side.
+
+You generally need to enter only one or both of the following in the form:
+
+* `GIT_REMOTE_REF`: Change this to refs/pull/NNNNN/head where NNNNN is the pull
+request ID.
```

Comment:
This parameter is also mentioned in L194-L198 above (the paragraph preceding "Useful CI Jobs"). Perhaps consolidate it into this section? And maybe head this section something like "Starting a CI job"?

Reply:
Another try